src/main/java/com/leetcode/practice/easy/binary_search/BinarySearch.java

```diff
@@ -0,0 +1,27 @@
+package com.leetcode.practice.easy.binary_search;
```

Comment:
Why package name have binary_search in it when all the problems of binary search are going to stay in BinarySearch class?